src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -33,7 +33,7 @@ trait PrestaShopTranslatorTrait
     public static $regexClassicParams = '/%\w+%/';
 
     /**
-     * {@inheritdoc}
+     * @todo: add the right PHPDoc
```

Comment:
as this trait doesn't extends the Translator class, even if it's used on Translator class only we can't use inheritdoc notation...